driver-core/src/test/functional/com/mongodb/internal/operation/FindOperationSpecification.groovy

```diff
@@ -672,7 +672,6 @@ class FindOperationSpecification extends OperationFunctionalSpecification {
         def operation = new FindOperation<Document>(getNamespace(), new DocumentCodec())
                 .projection(new BsonDocument('x', new BsonInt32(1)))
                 .filter(new BsonDocument('z', new BsonString('val')))
-                .allowDiskUse(true)
```

Comment:
Why was this removed?

Reply:
The `allowDiskUse` options can no longer be used with the wire version used in the test:
`java.lang.IllegalArgumentException: allowDiskUse not supported by wire version: 2`
The wire version must be 5 or higher for `allowDiskUse` to be supported.

Reply:
Right, but this is more than just a test of allowDiskUse, and the rest of it should be tested with 3.2. Just add another test case for allowDiskUse that only runs on 3.4+